docs/modules/ROOT/pages/index.adoc

```diff
@@ -785,6 +787,12 @@ There are a few possible relationships between the implementation of the *<<_ide
 
 IMPORTANT: The trust decisions described in the scenarios should only be evaluated once the *<<_identity_assertion,identity assertion>>* and the signature material within have been successfully validated as described in xref:_validating_the_identity_assertion[xrefstyle=full].
 
+The trust decision can result in one of three descriptions of the _<<_named_actor,named actor>>:_
+
+* *Trusted:* The _<<_identity_asertion_consumer,identity assertion consumer>>_ can verify a direct or indirect trust relationship to the _<<_named_actor,named actor>>_ through its established roots of trust.
+* *Well-formed:* The _<<_identity_asertion_consumer,identity assertion consumer>>_ can not verify a direct or indirect trust relationship to the _<<_named_actor,named actor>>._ 
+* *Revoked:* The credential used for signing the *<<_identity_assertion,identity assertion>>* has been revoked by the credential’s issuer.
```

Comment:
I'm not sure this is exclusive to *Trusted*.
If the consumer can verify the relationship, but the credential was revoked (either before or after the identity assertion was created) then would the description be *Trusted but Revoked*?